### ISSUE TITLE
Dependencies of record-extra v2.0.3 is newer than v3.0.0.

### ISSUE THREAD

**Issue by oreshinya** [justinwoo/purescript-record-extra#22]
record-extra latest (v3.0.0) depends on typelevel-prelude `^v4.0.0`, so compiling with latest compiler is failed. 

(But v2.0.3 depends on typelevel-prelude `>=4.0.0 <6.0.0`.)

```
[1/1 ExportConflict] bower_components/purescript-typelevel-prelude/src/Type/Row.purs:4:5

  4    , module Prim.RowList
         ^^^^^^^^^^^^^^^^^^^
  
  Export for type Prim.RowList.Cons conflicts with type class Prim.Row.Cons
```

Would you release new version?

**Comment by justinwoo** [comment]
Sure, sounds fine

**Comment by justinwoo** [comment]
Released 3.0.1, builds on 13.3

**Comment by oreshinya** [comment]
Thanks.
